From 5d61ca94c05561aefb11dfab4e15682c249c3746 Mon Sep 17 00:00:00 2001
From: Sean Parkinson <sean@wolfssl.com>
Date: Mon, 18 Nov 2024 08:29:16 +1000
Subject: [PATCH] KeyShareEntry: include private key fields for KYBER

Originallt HAVE_PQC and then changed to HAVE_FALCON and HAVE_DILITHIUM.
The KEM PQC algorithm is actually KYBER.
---
 wolfssl/internal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wolfssl/internal.h b/wolfssl/internal.h
index 70c99818e5..2519f09d8c 100644
--- a/wolfssl/internal.h
+++ b/wolfssl/internal.h
@@ -3539,7 +3539,7 @@ typedef struct KeyShareEntry {
     word32                keyLen;    /* Key size (bytes)                  */
     byte*                 pubKey;    /* Public key                        */
     word32                pubKeyLen; /* Public key length                 */
-#if !defined(NO_DH) || defined(HAVE_FALCON) || defined(HAVE_DILITHIUM)
+#if !defined(NO_DH) || defined(WOLFSSL_HAVE_KYBER)
     byte*                 privKey;   /* Private key - DH and PQ KEMs only */
     word32                privKeyLen;/* Only for PQ KEMs. */
 #endif
 